pack/fhv/fhvc.json

```diff
@@ -146,15 +146,296 @@
+    "back_name": "Atwood Farm",
+    "clues": 1,
+    "code": "10634",
+    "double_sided": true,
+    "encounter_code": "the_longest_night",
+    "encounter_position": 9,
+    "faction_code": "mythos",
+    "flavor": "The barn looks like it could collapse at any moment.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Barn",
+    "pack_code": "fhvc",
+    "position": 134,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "Enemies at Barn get +1 damage value.\nWhen Barn is revealed: Place the set-aside Ajax story asset at this location <b>([codex] Ω)</b>.",
+    "traits": "Farm.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "The old farm is riddled with scraps, desiccated livestock, rotten feed, and rusted farming equipment.",
+    "back_name": "Atwood Farm",
+    "clues": 1,
+    "code": "10635",
+    "double_sided": true,
+    "encounter_code": "the_longest_night",
+    "encounter_position": 10,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "Pasture",
+    "pack_code": "fhvc",
+    "position": 135,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "[free] During your turn, if Pasture has a trap on it: <b>Move</b> <i>(to a connecting location)</i>.\n<b>Forced</b> – After a ready, unengaged enemy moves into Pasture via a keyword, if there are no barriers at Pasture: Resolve that keyword again.",
+    "traits": "Farm.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n<cite>H.P. Lovecraft, \"The Colour Out of Space\"</cite>",
+    "clues": 3,
+    "code": "10636",
+    "double_sided": true,
+    "encounter_code": "the_longest_night",
+    "encounter_position": 11,
+    "faction_code": "mythos",
+    "illustrator": "Nele Diel",
+    "name": "Outer Fields",
+    "pack_code": "fhvc",
+    "position": 136,
+    "quantity": 1,
+    "shroud": 2,
+    "subname": "Bloodied Path",
+    "text": "[action]: Move an enemy to a connecting location. (Group limit once per game.)\n[reaction] After you discover the last clue from this location: Heal 2 damage from The Captives. (Group limit once per game.)",
+    "traits": "Field. Blight.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n<cite>H.P. Lovecraft, \"The Colour Out of Space\"</cite>",
+    "clues": 3,
+    "code": "10637",
+    "double_sided": true,
+    "encounter_code": "the_longest_night",
+    "encounter_position": 12,
+    "faction_code": "mythos",
+    "illustrator": "Nele Diel",
+    "name": "Outer Fields",
+    "pack_code": "fhvc",
+    "position": 137,
+    "quantity": 1,
+    "shroud": 2,
+    "subname": "Desolate Hills",
+    "text": "[free] During your turn: Look at the top 1 [per_investigator] cards of the Enemy deck. Discard any number of them and put the rest back on top in any order. (Group limit once per game.)",
```

Comment:
We prefer `[fast]` to `[free]` on the DB, please.

Reply:
Will do - this also needs a cleanup then, it's used a lot on other packs.